src/utils/components/InstanceTypeModal/StandaloneInstanceTypeModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+
+import useInstanceTypesAndPreferences from '@catalog/CreateFromInstanceTypes/state/hooks/useInstanceTypesAndPreferences';
+
+import ErrorAlert from '../ErrorAlert/ErrorAlert';
+import Loading from '../Loading/Loading';
+
+import { InstanceTypeModalProps } from './utils/types';
+import InstanceTypeModal from './InstanceTypeModal';
+
+const StandaloneInstanceTypeModal: FC<
+  Pick<InstanceTypeModalProps, 'isOpen' | 'onClose' | 'onSubmit' | 'vm'>
+> = ({ vm, ...otherProps }) => {
+  const { allInstanceTypes, loaded, loadError } = useInstanceTypesAndPreferences();
+  const instanceType = allInstanceTypes.find(
```

Comment:
shouldnt this be on useMemo? as u are passing all items each rendering

Reply:
IMHO it makes no difference. It could be even slower  - `allInstanceTypes` is less then 20 items. We can add it if you prefer.

Reply:
They are going to introduce new families of IT - so better include it as potential this number can be much larger